risc0/cpp/Makefile

```diff
@@ -0,0 +1,123 @@
+
+ifndef RISC0_CPP_TOOLCHAIN_DIR
+ifeq ($(shell command -v rzup 2>/dev/null),)
+  $(error "can't find rzup is not installed, I need it to find the risc0 cpp toolchain")
+endif
+RISC0_CPP_TOOLCHAIN_DIR := $(shell rzup show 2>/dev/null | grep -A1 "cpp" | grep -E "^\s*\*" | sed 's/.*\*\s*//' | head -1 | xargs -I {} find ~/.risc0 -name "*{}-cpp-*" -type d 2>/dev/null | head -1)/riscv32im-*/bin
```

Comment:
I think this logic could be removed, it's not working for me (I have no `rzup` and don't know how to install it). We could just assume risc0 is in `PATH`. I think it's the problem from whoever is compiling to install risc0 toolchain on the system and adjust `PATH` env.